### PR TITLE
setup.py, requirements.txt, README.md improvements

### PR DESCRIPTION
**Pull request by pietrodn**
This PR contains various improvements to setup.py, requirements.txt and README.md.

- future is not a direct dependency of dronekit: remove it (it will be installed by pymavlink)
- `README.md`: Python 3-compatible "Hello World"
- PyPI badge: link to the package page on PyPI
- `setup.py`: add classifier for Python 3
- `setup.py`: automatically discover packages with `setuptools` (subpackages of the `test` package have been missing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ The [Quick Start](http://python.dronekit.io/guide/quick_start.html) guide explai
...
 
 ```python
-from dronekit import connect, VehicleMode
+from dronekit import connect
 
 # Connect to UDP endpoint.
 vehicle = connect('127.0.0.1:14550', wait_ready=True)
 # Use returned Vehicle object to query device state - e.g. to get the mode:
-print " Mode: %s" % vehicle.mode.name
+print("Mode: %s" % vehicle.mode.name)
```
Comment (hamishwillee) [inline README.md]: Does this actually work without importing VehicleMode?

File: setup.py [hunk 2]
```diff
@@ -1,32 +1,30 @@
-from setuptools import setup, Extension
-import platform
+import setuptools
 
 version = '2.9.1'
```
Comment (hamishwillee) [inline setup.py]: Should version be updated?

File: README.md (hunk 1 above)
Comment (pietrodn) [reply README.md]: Yes. I just ran this test in SITL and it works as expected.
```python
from dronekit import connect
from dronekit.test import with_sitl


@with_sitl
def test_readme(connpath):
    # Connect to UDP endpoint.
    vehicle = connect(connpath, wait_ready=True)
    # Use returned Vehicle object to query device state - e.g. to get the mode:
    print("Mode: %s" % vehicle.mode.name)
    vehicle.close()
```

Output:
```
$ nosetests -svx dronekit.test.sitl.test_readme
dronekit.test.sitl.test_readme.test ... Mode: STABILIZE
ok

----------------------------------------------------------------------
Ran 1 test in 16.342s

OK
```

File: setup.py (hunk 2 above)
Comment (pietrodn) [reply setup.py]: Probably yes, when a new release is made. But that's your call :-)

Comment (hamishwillee) [reply setup.py]: :-) @peterbarker 's

**Comment by pietrodn** [comment]
Is there any further issue with this PR?

**Comment by hamishwillee** [comment]
I have no issue with it - but would hope @peterbarker would confirm?

**Comment by mrpollo** [comment]
I'll defer to @peterbarker since I haven't been very involved with the project and I don't know the current status, but I can help merge if no one has time to review.

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**